### PR TITLE
Issue 79 Add the letter to all members (md)

### LINKED ISSUE #79: [媒体文章搬到社区网站] 文案1：致 wuhan2020 开源社区全体成员的一封信
https://mp.weixin.qq.com/s/jFBpUFnSwop4PCGd_5lcWA

标题作者请写媒体和作者名字
请使用 [Typora](https://www.typora.io/)  将`html`页面转化为`markdown`,放到社区频道.图片尽量下载到本站

### PR DESCRIPTION
**Pull request by swmiao**
fixes https://github.com/wuhan2020/wuhan2020.github.io/issues/79

### CODE REVIEW

**Review by lovepoem — CHANGES_REQUESTED** [review]
1. toAllMembers.md should be to-all-members.md

2.  the `.md` file should be  placed in `/blog/zh-cn`
3. Add a config  in the file `site_config/community.jsx`

 
image

File: toAllMemebers.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
...////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////wAAAP///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////wAAAP///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////wAAAP///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////wAAAP///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////wAAAP///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////wAAAP///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////wAAAA==)
```
Comment (GindaChen) [inline toAllMemebers.md]: I can't see this image when I open markdown, but I can open the link and see the QR code.

**Review by GindaChen — CHANGES_REQUESTED** [review]

**Comment by GindaChen** [comment]
Need English translation?

**Comment by lovepoem** [comment]
> Need English translation?

Should contact with @ch-shumin

**Review by GindaChen — APPROVED** [review]

**Pull request merged**